test/e2e/flowexporter_test.go

```diff
@@ -103,17 +142,74 @@ func TestFlowExporter(t *testing.T) {
 		t.Fatalf("Error when waiting for network policy to be realized: %v", err)
```

Comment:
deployNetworkPolicy function in the first commit is good. However, defer function that deletes the network policies needs to be in the Test* function.